ui/public/config.json

```diff
@@ -56,5 +56,6 @@
   "plugins": [],
   "basicZoneEnabled": true,
   "multipleServer": false,
+  "loginFooter": "",
```

Comment:
Isn't up with the main ui footer text, a more obvious place to put this setting in the config.json file?

```
  "docBase": "http://docs.cloudstack.apache.org/en/latest",
  "appTitle": "CloudStack",
  "footer": "Licensed under the <a href='http://www.apache.org/licenses/' target='_blank'>Apache License</a>, Version 2.0.",
  "loginFooter": "",
  "logo": "assets/logo.svg",
  
```